linux/shim/service.go

```diff
@@ -211,6 +212,7 @@ func (s *Service) Resume(ctx context.Context, r *shimapi.ResumeRequest) (*google
 }
 
 func (s *Service) Exit(ctx context.Context, r *shimapi.ExitRequest) (*google_protobuf.Empty, error) {
+	unix.Unmount(filepath.Join(s.path, "rootfs"), 0)
```

Comment:
idk, we don't have a way to build on darwin or whatever right now so I don't know what will break with that changes.  I think if we want to continue supporting it we need to get builds on the CI //cc @justincormack